osquery/tables/system/windows/intel_me.cpp

```diff
@@ -124,31 +125,65 @@ void getHECIDriverVersion(QueryData& results) {
                         nullptr);
 
   if (ret == 0) {
-    VLOG(1) << "Device IOCTL call failed with " << GetLastError();
+    auto last_error = GetLastError();
+    if (last_error == ERROR_GEN_FAILURE) {
+      VLOG(1) << "The driver is already in use by another client and can't be "
+                 "queried at this time";
+    } else {
+      VLOG(1) << "Device IOCTL call failed with " << last_error;
+    }
+
     CloseHandle(driver);
     return;
   }
 
+  if (response.version != 0x1) {
```

Comment:
Does this help in fixing the issue?

Reply:
Lets do it in separate PR if it's not about this issue.

Reply:
This is a sanity check also performed by the linux code. The structure layout we are using to read the version information is tied to the version we are expecting; without it we risk not noticing a structure change, returning wrong data in the table.

Should we remove it from this PR?

Reply:
It will help. Let's just add bare minimum to solve the issue. This might be the issue by itself but as I understand the separate one.

Reply:
I've removed the version check as you asked :+1: